cogs/message_counter.py

```diff
@@ -49,7 +46,9 @@ async def on_message(self, message: commands.Context) -> None:
             messages_sent = messages_sent[0]
 
         # If the message was sent in a help channel, update the helpMessagesSent column
-        if "help" in category_name:
+        if (
+            category := getattr(message.channel, "category")
```

Comment:
`getattr` is used to access attributes dynamically, so if the attribute is defined as a constant, it is no safer than a typical property access
```suggestion
            category := message.channel.category
```

Reply:
Good point - I forgot to add a default value to the `getattr` call. Fixed in 1d94d84bac48df0f537c00e887c4690595ffe923.
([discussed in discord](https://discord.com/channels/238956364729155585/1040360228388098159/1145473041984389130))